### PR TITLE
Update for astropy 1.0, Python 3, and generally refresh

### PR DESCRIPTION
**Pull request by taldcroft**
@astrofrog @hamogu @DougBurke 

I took a crack at giving Python4Astronomers a long-needed refresh:

- Python 3 compatible where possible
- Remove deprecated / unsupported packages like PyFITS etc
- Move VO tutorial out of the main lineup since it depends heavily on orphaned packages
- Change from `ipython --pylab` to `ipython --matplotlib` everywhere.
- Update comments, recommendations, style.
- Remove ATpy section (sorry @astrofrog)

### CODE REVIEW

**Comment by taldcroft** [comment]
And I think it's time for a new theme, which should look familiar!

image

**Comment by taldcroft** [comment]
@astrofrog - Do you know what sort of attribution should be included for using the astropy-bootstrap theme?

**Comment by taldcroft** [comment]
Preview at http://python4astronomers.github.io/python4astronomers/site/

**Comment by astrofrog** [comment]
Great! I was thinking an update was overdue, so thanks for putting in the time to do it!

I can review this in detail later this week, but +1 on removing ATpy of course, since it's deprecated in favor of ``astropy.table`` (I'm only keeping the package on PyPI for legacy purposes).

Not sure about the attribution for astropy-bootstrap

File: source/installation/python_install.rst [hunk 1]
```diff
@@ -82,7 +87,6 @@ installation was reported.
   pip install --upgrade aplpy
```
Comment (astrofrog) [inline source/installation/python_install.rst]: For the line above, pip install astropy not needed if using conda

File: source/astropy/tables.rst [hunk 2]
```diff
@@ -17,7 +17,7 @@ Documentation
 -------------
 
 For more information about the features presented below, you can read the
-`astropy.table <http://docs.astropy.org/en/v0.2.1/table/index.html>`_ docs.
+`astropy.table <http://astropy.readthedocs.org/en/stable/table/index.html>`_ docs.
```
Comment (astrofrog) [inline source/astropy/tables.rst]: I think this should be docs.astropy.org/en/stable/table/index.html

File: source/fitting/fitting.rst [hunk 3]
```diff
@@ -15,6 +15,10 @@ Workshop goals are to use Sherpa to:
 
 - Fit the MAST 3C 273 data using the low-level API
 
+.. Note::
+
+    Sherpa currently does not support Python 3.
```
Comment (astrofrog) [inline source/fitting/fitting.rst]: :cry:

**Comment by astrofrog** [comment]
@taldcroft - this looks great! Just a couple of minor comments above then this should be merged.

File: source/installation/python_install.rst (hunk 1 above)
Comment (taldcroft) [inline source/installation/python_install.rst]: @astrofrog - did you misread this?  It already says that pip install is not need for astropy, whereas you are commenting on the `aplpy` install line.

**Pull request merged**